cinnabar/femap.py

```diff
@@ -65,19 +68,20 @@ class FEMap:
     >>> # Load/create experimental results
     >>> from openff.units import unit
     >>> kJpm = unit.kilojoule_per_mole
-    >>> experimental_result1 = AbsoluteMeasurement(label="CAT-13a", DG=-8.83 * kJpm, uncertainty=0.10 * kJpm)
-    >>> experimental_result2 = AbsoluteMeasurement(label="CAT-17g", DG=-9.73 * kJpm, uncertainty=0.10 * kJpm)
+    >>> g = GroundState()
```

Comment:
Could you introduce the idea of label here so that folks can understand that it can be set to different states that hash differently?